packages/cli/src/util/logger.js

```diff
@@ -23,13 +23,32 @@ const error = (...messages: Array<string>) => {
   );
 };
 
+const logError = (err: Error) => {
+  console.error(`${chalk.black.bgRed(' ERROR ')} ${chalk.red(err.message)} \n`);
+  if (err instanceof ProcessError) {
+    console.error(`${chalk.grey(err.processError)}`);
+  }
+};
+
 const debug = (...messages: Array<string>) => {
   console.log(`${chalk.black.bgWhite(' DEBUG ')} ${joinMessages(messages)}`);
 };
 
+class ProcessError extends Error {
```

Comment:
This doesn't belong to logger. Can you move it to some kind of `errorHelpers`?

Reply:
Yup, good idea. I'd suggest "errors" for now.